public/js/reader.js

```diff
@@ -438,21 +424,24 @@ Reader.goToPage = function (page) {
         if (img1.naturalWidth > img1.naturalHeight || img2.naturalWidth > img2.naturalHeight) {
             // Depending on whether we were going forward or backward, display img1 or img2
             $("#img").attr("src", Reader.previousPage > Reader.currentPage ? img2.src : img1.src);
-            $("#img_pre").attr("src", "");
+            $("#img_doublepage").attr("src", "");
+            $("a#display").removeClass("double-mode");
             Reader.showingSinglePage = true;
         } else {
             if (Reader.mangaMode) {
                 $("#img").attr("src", img2.src);
-                $("#img_pre").attr("src", img1.src);
+                $("#img_doublepage").attr("src", img1.src);
             } else {
                 $("#img").attr("src", img1.src);
-                $("#img_pre").attr("src", img2.src);
+                $("#img_doublepage").attr("src", img2.src);
             }
+            $("a#display").addClass("double-mode");
```

Comment:
There's only one `display` element, so is the `a` selector needed?
```suggestion
            $("#display").addClass("double-mode");
```